packages/renderer/src/lib/troubleshooting/TroubleshootingPageStore.svelte

```diff
@@ -19,27 +21,23 @@ let openDetails = false;
 
 <div class="flex flex-col bg-charcoal-800 p-2 items-center rounded w-full">
   <div><svelte:component this="{eventStoreInfo.iconComponent}" size="20" /></div>
-  <div class="text-xl">
-    <button
-      disabled="{fetchInProgress}"
-      class="underline outline-none"
-      title="Open Details"
-      aria-label="Open Details"
-      on:click="{() => (openDetails = true)}">
-      {eventStoreInfo.name}
-    </button>
-  </div>
+  <Button
+    bind:disabled="{fetchInProgress}"
+    class="underline"
```

Comment:
not sure why we need that if we have type=link ?

shouldn't it already formatted correctly ?

Reply:
I didn't know these titles were clickable until I saw the code, and updating to our Button design meant the underline was only visible on hover which made it worse. So, I thought I would at least leave the underline for now.

I talked to Mairin briefly and she had some ideas how these links could be redesigned, but these are the one button on the page that aren't directly using PF classes nor copying the same design, so the simplest answer for this PR is to leave them as-is and handle another time. I've removed the changes to this 'button' in an updated commit.